techmatters/flex-plugins
# Chi 1436 refactor case saving logic redux promise

Primary reviewer: @stephenhand 

## Description


**Feature Update**: Refactor case saving logic to use redux promise

**Goal**: To refactor the current saving logic to be triggered in the redux layer off the back of a save action, not in the component.

### Checklist

### Related Issues
Fixes [CHI-1436](https://tech-matters.atlassian.net/browse/CHI-1436)

### Verification steps

- Go to Case List
- Select a case
- Edit case summary
- Verify that that update was successful
- Go to Offline Contacts
- Fill out the forms
- Click "Add Contact to New Case"
- Verify that contact was successfully added to new case


[CHI-1436]: https://tech-matters.atlassian.net/browse/CHI-1436?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
